libs/api/src/base.ts

```diff
@@ -8,12 +8,9 @@ export const OkResponseSchema: z.ZodSchema<OkResponse> = z.object({
   status: z.literal('ok'),
 });
 
-export interface ErrorsResponse {
-  status: 'error';
-  errors: Array<{ type: string; message: string }>;
-}
-
-export const ErrorsResponseSchema: z.ZodSchema<ErrorsResponse> = z.object({
+export const ErrorsResponseSchema = z.object({
```

Comment:
had to take out the template param and let type inference do its thing to get the `discriminatedUnion` working - but, let me know if this pattern of deriving the response type from the schema breaks anything (functional or ideological)